helix-core/src/main/java/org/apache/helix/controller/stages/CustomizedViewAggregationStage.java

```diff
@@ -112,14 +115,18 @@ public void execute(final ClusterEvent event) throws Exception {
       }
       // add/update customized-views from zk and cache
       if (updatedCustomizedViews.size() > 0) {
-        dataAccessor.setChildren(keys, updatedCustomizedViews);
+        boolean[] success = dataAccessor.setChildren(keys, updatedCustomizedViews);
```

Comment:
Can we log somewhere how many updates are failed? It's useful information.

Reply:
I added a logging in CustomizedViewMonitor because it is already looping through this boolean[] so it's the most convenient place to add it.